spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/vectorstore/gemfire/GemFireVectorStoreAutoConfiguration.java

```diff
@@ -57,7 +57,7 @@ public GemFireVectorStore gemfireVectorStore(EmbeddingModel embeddingModel, GemF
 			.setVectorSimilarityFunction(properties.getVectorSimilarityFunction())
 			.setFields(properties.getFields())
 			.setSslEnabled(properties.isSslEnabled());
-		return new GemFireVectorStore(config, embeddingModel);
+		return new GemFireVectorStore(config, embeddingModel, properties.isInitializeSchema());
```

Comment:
I noticed now that there is duplication between `GemFireVectorStoreConfig` and `GemFireVectorStoreProperties`.  There should only be one class for configuration, `GemfireVectorStoreProperties`  Use of `GemFireVectorStoreConfig` should be removed.  I will be merging the initialization logic update in this PR as part of in the lager schema initialization PR.  Please make a followup PR to remove this duplication